next-tick.js

```diff
@@ -0,0 +1,5 @@
+module.exports = nextTick;
```

Comment:
```suggestion
module.exports = process.nextTick;
```

Reply:
In this case it doesn't pass [this test](https://github.com/nodejs/readable-stream/blob/master/test/ours/lolex-fake-timers.js).

Reply:
the best approach is to use `module.exports.nextTick = process.nextTick` pattern instead. Adding a wrapper is going to slow things down.

Reply:
As i told you above it won't pass [this test](https://github.com/nodejs/readable-stream/blob/master/test/ours/lolex-fake-timers.js) if I use your approach.

Reply:
Oh, I understand now. You can do the following on the node side:

```js
module.exports = process
```

While on the browser side:


```js
module.exports = { nextTick }
```

Then, in code:

```js
var wrap = require('./nextTick')
wrap.nextTick(function () {
  // this will work via lolex
})
``

Reply:
Well, it's basically a polyfilling of a `process` object which I suggested here: https://github.com/nodejs/readable-stream/pull/413#issuecomment-508768361

Reply:
I think we were in agreement in there. I'll just not call it `process`.

Reply:
But how do you want it to be called? `wrap`?

Reply:
We should not expect to be a full `process` object. In the browser, it won't be. It's an object with a `nextTick` property. we can call how we want "lightProcess" "fakeProcess" "nextTickWrap". This should enable that lolex support.